pyabc/visualization/epsilon.py

```diff
@@ -1,56 +1,53 @@
 import matplotlib.pyplot as plt
 import matplotlib as mpl
+import matplotlib.axes
 from matplotlib.ticker import MaxNLocator
 from typing import Union, List
 import numpy as np
 
 from ..storage import History
-from .util import to_lists_or_default
+from .util import to_lists, get_labels
 
 
 def plot_epsilons(
         histories: Union[List, History],
         labels: Union[List, str] = None,
         colors: List = None,
-        scale: str = None,
+        yscale: str = 'log',
         title: str = "Epsilon values",
         size: tuple = None,
-        ax: mpl.axes.Axes = None):
+        ax: mpl.axes.Axes = None) -> mpl.axes.Axes:
     """
     Plot epsilon trajectory.
 
     Parameters
     ----------
-
-    histories: Union[List, History]
+    histories:
         The histories to plot from. History ids must be set correctly.
-    labels: Union[List ,str], optional
+    labels:
```

Comment:
why the datatype + default value are removed?

Reply:
We have datatype annotations now always (mostly) directly in the python code. Having it in the docstring as well is thus superfluous, and potentially leads to problems if both are not synchronized.

Reply:
totally agreed.